src/core.js

```diff
@@ -172,6 +172,7 @@ extend( QUnit, {
 } );
 
 QUnit.pushFailure = pushFailure;
+QUnit.pushFailureAndResume = pushFailureAndResume;
```

Comment:
It's a really verbose name. I'd like to see something shorter, but won't stand too firmly on it.